trade-finance-manager-api/src/v1/emails/AIN-MIN-confirmation/gef-facilities-list.js

```diff
@@ -16,71 +16,58 @@ const mapIssuedValue = (hasBeenIssued) => {
   return CONSTANTS.FACILITIES.FACILITY_STAGE_PORTAL.UNISSUED;
 };
 
+/**
+ * @param {unknown} value
+ * @returns {string | undefined} Returns 'Yes'/'No' if value is True/False, or undefined otherwise
+ */
+const mapBooleanToYesOrNo = (value) => {
+  if (typeof value === 'boolean') {
+    return value ? 'Yes' : 'No';
+  }
+  return undefined;
+};
+
 /*
  * facilityFieldsObj
  * returns a new object with the fields/values we need to consume and display in the email.
  */
-const facilityFieldsObj = (facility) => {
-  const fields = (({
-    ukefFacilityId,
-    bankReference,
-    hasBeenIssued,
-    coverStartDate,
-    coverEndDate,
-    value,
-    currencyCode,
-    coverPercentage,
-    interestPercentage,
-    guaranteeFee,
-    ukefExposure,
-    feeType,
-    feeFrequency,
-    dayCountBasis,
-  }) => ({
-    ukefFacilityId,
-    bankReference,
-    hasBeenIssued,
-    coverStartDate,
-    coverEndDate,
-    value,
-    currencyCode,
-    coverPercentage,
-    interestPercentage,
-    guaranteeFee,
-    ukefExposure,
-    feeType,
-    feeFrequency,
-    dayCountBasis,
-  }))(facility);
-
-  // format for emails
-  if (fields.hasBeenIssued) {
-    fields.hasBeenIssued = mapIssuedValue(fields.hasBeenIssued);
-  }
-
-  if (fields.coverStartDate) {
-    fields.coverStartDate = format(Number(fields.coverStartDate), 'do MMMM yyyy');
-  }
-
-  if (fields.coverEndDate) {
-    const epochCoverEndDate = new Date(fields.coverEndDate);
-    fields.coverEndDate = format(Number(epochCoverEndDate), 'do MMMM yyyy');
-  }
-
-  if (fields.coverPercentage) {
-    fields.coverPercentage = `${fields.coverPercentage}%`;
-  }
-
-  if (fields.interestPercentage) {
-    fields.interestPercentage = `${fields.interestPercentage}%`;
-  }
-
-  if (fields.guaranteeFee) {
-    fields.guaranteeFee = `${fields.guaranteeFee}%`;
-  }
-
-  return fields;
-};
+const facilityFieldsObj = ({
+  ukefFacilityId,
+  bankReference,
+  hasBeenIssued,
+  coverStartDate,
+  coverEndDate,
+  isUsingFacilityEndDate,
+  facilityEndDate,
+  bankReviewDate,
+  value,
+  currencyCode,
+  coverPercentage,
+  interestPercentage,
+  guaranteeFee,
+  ukefExposure,
+  feeType,
+  feeFrequency,
+  dayCountBasis,
+}) => ({
+  ukefFacilityId,
+  bankReference,
+  hasBeenIssued: hasBeenIssued && mapIssuedValue(hasBeenIssued),
+  coverStartDate: coverStartDate && format(Number(coverStartDate), 'do MMMM yyyy'),
+  coverEndDate: coverEndDate && format(parseISO(coverEndDate), 'do MMMM yyyy'),
+  isUsingFacilityEndDate: mapBooleanToYesOrNo(isUsingFacilityEndDate),
+  facilityEndDate: facilityEndDate && format(parseISO(facilityEndDate), 'do MMMM yyyy'),
+  bankReviewDate: bankReviewDate && format(parseISO(bankReviewDate), 'do MMMM yyyy'),
```

Comment:
Are we able to use constants for the date formats? We have `LONG_FORM_DATE` in portal